client/webserver/site/src/js/markets.ts

```diff
@@ -557,6 +559,37 @@ export default class MarketsPage extends BasePage {
         setPriceAndChange(s.tmpl, xc, mkt)
       }
     }
+    this.page.rateField.value = fourSigFigs(this.defaultRate())
```

Comment:
I think this just sets "default" value for rate when market page loads or re-loads (based on current price),
actually I'm already doing it in `resolveOrderFormVisibility`, so I don't need it here, removed.

> In fact, we should never be setting the rate field for the user.

or do you mean it like [no defaults for lots/quantity](https://github.com/decred/dcrdex/pull/1921#discussion_r1011652574) we discussed above ? I think almost all exchanges suggest some sort of default price, and that's very convenient feature IMO because typing in decimal point value with lots of digits is error-prone and pretty much unnecessary burden on the user (compared to suggested one). I can see how having defaults for lots/quantity could be problematic (having all fields: rate, lots/quantity set allows for placing order  mistakenly, "without looking"), but since we don't set lots/quantity defaults now I don't see how setting default rate could result into issues, so why not ?

Reply:
Mmkay. I think you convinced me.